free-1.md

```diff
@@ -11,41 +11,100 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+The need is to improve the ease of creating crochet patterns, and to allow
+people who are currently working on a crochet project to easily keep track of
+the progress that they have made on a project.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+In terms of creating a crochet pattern, it would be very benificial for a
+pattern creater to have a programming language where they could easily add to,
+and manipulate the pattern, and then have an easy way to create a beautiful
+pattern output. Then, for someone using the pattern it would be easy
+to follow. There could be both a PDF output and an interative way to state
+what progress you have made in the project while following the pattern. You
+could also add your own notes, or edit the pattern yourself using the language.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I like to crochet, and I find it difficult to keep track of where I am in my
+project when I am either following a pattern, partially following a pattern
+while making my own changes, or when I have created my own projects without a
+pattern. I came up with this idea because I would like there to be an easy way
+to create crochet patterns, and I am already knowledgeable about this domain. It
+is also a cool domain because it already has it's own abbreviations (it's own
+language) that could be used in the language.
+
```

Comment:
I can definitely relate with this; sometimes I start knitting something and then I leave it for 3+ months. When I come back, I have no clue what kind of stitches I was doing, nor where I was in the pattern. Perhaps you address this later but it would be convenient to be able to have multiple project files to account for the fact that the unorganized (like me) usually have many work-in-progress projects at a time.